oracle/provider/price_store.go

```diff
@@ -114,7 +114,7 @@ func (ps *priceStore) GetTickerPrices(pairs ...types.CurrencyPair) (types.Curren
 		key := ps.currencyPairToTickerPair(cp)
 		ticker, ok := ps.tickers[key]
 		if !ok {
-			ps.logger.Warn().Msgf("failed to get ticker price for %s", key)
```

Comment:
I think all of these are great change except this one. If we are missing ticker prices for a single provider/pair it is something we should eventually fix but not something that impedes the price feeder from operating normally. If it causes an asset to be unavailable then we have the "unable to report price for expected asset" you escalated to error https://github.com/ojo-network/price-feeder/pull/245/files#diff-cea4d8e05c26d6e61c0f334cce97b05da5115d25082ab0f21b95548edac0dbdfR261